.github/actions/publish/action.yml

```diff
@@ -113,6 +119,7 @@ runs:
             PATH="/c/Program Files (x86)/Windows Kits/10/bin/${runner_arch}:${PATH}"
         fi
 
+        npm i --no-save oclif
```

Comment:
The reason this is here and not in a devDependency is because: https://github.com/oclif/oclif/issues/1324 and shrinkwrap is included in the final bundle, so this avoids +5MB on the final bundle and our CI breaking because the file ends up being too large, it can go into devDependency oclif @aws-sdk is made optional